package.json

```diff
@@ -18,14 +18,15 @@
     "lint": "eslint .",
     "test": "node test.js",
     "posttest": "npm run lint",
-    "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require test/init.js test/*.test.js"
+    "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require loopback-connector-couchdb2/test/interfaces/lb-bdd.js --require test/init.js --require test/*.test.js --ui lb-bdd"
```

Comment:
I think after we release the new lb-bdd repo we can change the mocha cmd here. Depends on which PR landed first.